domain-cc/cc-app/src/python_src/pydantic_models.py

```diff
@@ -27,3 +30,43 @@ class PredictedClassification(BaseModel):
 
     classification_code: int
     classification_name: str
+
+
+class Contention(BaseModel):
+    contention_text: str
+    diagnostic_code: Optional[int]  # only required for claim_type: "claim_for_increase"
+
+
+class ClaimType(str, Enum):
```

Comment:
Should this move to be part of the Contention class since a claim can have a mix of new and increase claims?